bin/controller.py

```diff
@@ -45,7 +46,7 @@ def post_new():
             code = forms.get('code', '').encode('latin-1') or code
             ext = parse_extension(forms.get('lang')) or ext
             maxusage = int(forms.get('maxusage') or maxusage)
-            lifetime = int(forms.get('lifetime') or lifetime)
+            lifetime = Time(forms.get('lifetime') or lifetime)
```

Comment:
Ici le lifetime par défaut est déjà un Time (voir config.py). De plus, pourquoi ne pas directement le convertir en int ici plutôt que dans la création du Snippet ?
```suggestion
            lifetime = int(Time(forms.get('lifetime')) or lifetime)
```

Reply:
C'est un choix de Lazor que je ne saurais pas vraiment expliqué. Mais a mon avis, étant donné que ca hérite de int, on a juste besoin de le cast pour expire

Reply:
Oui si on caste directement alors on perd le `__str__` custom des Time donc autant cast en int *uniquement* là où c'est nécessaire

Reply:
Ok, alors c'est bon pour moi 👌

Reply:
Et c'est vrai qu'on pourrait `Time(form.get(...)) or DEFAULT_LIFETIME` mais pour ce que ça change....

Reply:
(non ca fait une erreur)